specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2021-07-01/examples/Updates/List.json

```diff
@@ -0,0 +1,48 @@
+{
+  "parameters": {
+    "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "resourceGroupName": "System.Redmond",
+    "updateLocation": "Redmond",
+    "api-version": "2016-05-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/23d66fd1-4743-42ff-b391-e29dc51d799e/resourceGroups/System.Redmond/providers/Microsoft.Update.Admin/updateLocations/redmond/updates/Microsoft1.0.180302.1 ",
+            "name": "Microsoft1.0.180302.1 ",
+            "type": "Microsoft.Update.Admin/updateLocations/updates",
+            "location": "redmond ",
+            "tags": {},
+            "properties": {
+              "displayName": "displayName",
+              "description": "MAS Update 1.0.180302.1",
+              "version": "1.0.180302.1",
+              "availabilityType": "online",
+              "state": "Installed",
+              "releaseLink": "https://aka.ms/azurestackupdate",
+              "prerequisites": [],
+              "packageSizeInMb": 23345,
+              "packagePath": "\\\\SU1FileServer\\SU1_Infrastructure_2\\Updates\\Packages\\Microsoft1.0.180302.1",
+              "publisher": "Microsoft",
+              "updateStateProperties": {
+                "downloadPercentage": 100,
+                "installedDate": "2018-03-03T08:09:12.8824552Z"
+              },
+              "azsUpdateProperties": {
+                "minVersionRequired": "1.2008.0.43",
+                "minOemVersionRequired": "2.1.1908.0"
+              },
+              "oemUpdateProperties": {
+                "oemFamily": "HPE",
+                "oemNotifyMsg": "This is notify message."
```

Comment:
I think these should still be removed from the example since it's showing a hub update (MAS Update 1.0.180302.1) and should not have OEM fields. Should we include a second entry in the example to show an OEM update?

Reply:
Also I think Dan had a comment around updating the versions in the example. We should update the ones in the version, description, and packagePath properties for this mock object.

Reply:
We need a separate example for the oem update.

Reply:
Added more examples in "List", and made them more realistic.